arex-web-api/src/main/java/com/arextest/web/api/service/controller/ReportQueryController.java

```diff
@@ -378,7 +378,7 @@ public Response countRecord(@Valid @RequestBody CountRecordRequestType requestTy
     long currentTime = System.currentTimeMillis();
 
     if (requestType.getBeginTime() == null) {
-      requestType.setBeginTime(currentTime - 4 * 24 * 60 * 60 * 1000L);
+      requestType.setBeginTime(0L);
```

Comment:
ensure the usage of index in the method of "getLastRecordVersionMocker" and "mongoTemplate.count"
Test: the consumed time of querying on unusal suitation that the record case exists one year